hack/prerelease.sh

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+SCRIPT_DIR="$(
+    cd "$(dirname "$0")" >/dev/null
+    pwd
+)"
+
+PROJECT_DIR="$(
+    cd "$SCRIPT_DIR/.." >/dev/null
+    pwd
+)"
+
+usage() {
+    echo "
+Usage:
+    ${0##*/} [options]
+
+Optional arguments:
+    -p, --product $PRODUCT
+        The product on which to activate the pre-release subscription.
+        Can be specified multiple times.
+    -d, --debug
+        Activate tracing/debug mode.
+    -h, --help
+        Display this message.
+
+Example:
+    ${0##*/}
+" >&2
+}
+
+parse_args() {
+    PRODUCT_LIST=()
+    while [[ $# -gt 0 ]]; do
+        case $1 in
+        -p|--product)
```

Comment:
There's no check to verify that $2 exists when -p/--product is specified. This could cause an error if someone runs -p without a product name. 
```suggestion
        -p|--product)
            if [[ -z "${2:-}" ]]; then
                echo "[ERROR] Product name required for $1 option"
                usage
                exit 1
            fi
```

Reply:
Done